src/app/missing_blocks_auditor/missing_blocks_auditor.ml

```diff
@@ -134,8 +150,14 @@ let main ~archive_uri () =
         [%log info] "Length of canonical chain is %Ld blocks" chain_len
       else (
         add_error chain_length_error ;
-        [%log info] "Length of canonical chain is %Ld blocks, expected: %Ld"
-          chain_len highest_canonical ) ;
+        if genesis_or_fork_block_height = 1 then
+          [%log info] "Length of canonical chain is %Ld blocks, expected: %Ld"
+            chain_len highest_canonical
+        else
+          [%log info]
+            "Length of canonical chain is %Ld blocks, expected: %Ld. (Note: \
```

Comment:
I think in this case it shouldn't be genesis. So it's not "genesis or first fork block has height", but rather, "first fork block has height".